dev/WebView2/WebView2AutomationPeer.h

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "WebView2AutomationPeer.g.h"
+#include "WebView2.h"
+#include "UIAutomationCore.h"
+
+MIDL_INTERFACE("865F5B88-6506-4E64-A4C5-4B7723650731")
+IAutomationPeerHwndInterop : public IUnknown
+{
+public:
+    virtual /* [propput] */ HRESULT STDMETHODCALLTYPE GetRawElementProviderSimple(
+        /* [retval][out] */ _Outptr_opt_ IRawElementProviderSimple** value) = 0;
+
+    virtual /* [propget] */ HRESULT STDMETHODCALLTYPE IsCorrectPeerForHwnd(
+                            HWND hwnd,
+        /* [retval][out] */ _Out_ bool* value) = 0;
```

Comment:
`bool` is not an ABI type. Did you mean `BOOL` (int-sized) or `BOOLEAN` (think that's byte-sized)? What's the original header say?

Reply:
Oops, this is a WinUI3 thing. We're doing the UIA integration differently in WinUI2. Removing.